fdbserver/include/fdbserver/DataDistribution.actor.h

```diff
@@ -45,7 +45,17 @@
 // RelocateReason to DataMovementReason is one-to-N mapping
 class RelocateReason {
 public:
-	enum Value : int8_t { OTHER = 0, REBALANCE_DISK, REBALANCE_READ, MERGE_SHARD, SIZE_SPLIT, WRITE_SPLIT, __COUNT };
+	enum Value : int8_t {
+		OTHER,
+		REBALANCE_DISK,
+		MOVE_SHARD,
+		MULTIPLY_SHARD,
```

Comment:
I was thinking about this recently and I think it would be better to call this something more generic, like `ADD_REPLICAS` and `REDUCE_REPLICAS`.

Reply:
change `MULTIPLY_SHARD` to `ADD_SHARD`.